[15.0][ADD] account_move_search_line (#1504)
OCA/account-invoicing: account_move_search_line/readme/DESCRIPTION.rst

> This module provides to search with invoice line values in account invoice search views.

Reviewer: ```suggestion
This module allows for searching by invoice line values (labels) in the invoice search view.
```
